### PR TITLE
Improve g5k scripts

### PR DESCRIPTION
**Pull request by ergl**
General script cleanup, also simplified some parts.

I added some `todo`s in portions of the code I found confusing.

### CODE REVIEW

**Comment by bieniusa** [comment]
@aletomsic can you check that it fits your setup ideas? I am not using grid5k.

File: script/configMachines.sh [hunk 1]
```diff
@@ -18,17 +18,16 @@ AllCookies=`cat /root/basho_bench1/basho_bench/script/allcookiesfull`
 echo All nodes "$AllNodes"
 mkdir -p logs/"$JobId"
 
-File1="./antidote/rel/vars.config"
+File1="./antidote/rel/vars/dev_vars.config.src"
```
Comment (ergl) [inline script/configMachines.sh:21]: Actually, I'm not sure if this is the file that needs to be changed. I see there's another configuration file in `antidote/config/vars.config` with the same information.

**Comment by aletomsic** [comment]
Sure I’ll take a look.

**Comment by ergl** [comment]
Closing since we've been working on a new version of the deployment scripts

**Pull request closed without merging**